src/main/java/io/blamer/bot/conversation/Conversation.java

```diff
@@ -27,22 +27,23 @@
 import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
 import org.telegram.telegrambots.meta.api.objects.Update;
 import org.telegram.telegrambots.meta.api.objects.commands.BotCommand;
+import reactor.core.publisher.Mono;
 
 /**
  * This interface created for solution of infinity switch-case.
  * <pre>{@code
  * @Component("/example")
  * public class Example implements Conversation {
  *      @Override
- *      public SendMessage messageFromUpdate(final Update update) {
+ *      public SendMessage messageOf(final Update update) {
```

Comment:
@l3r8yJ since you update return type to `Mono`, we also need such update here

Reply:
@h1alexbel what do you mean?

Reply:
here is, in the javadoc return type is `SendMessage`, while real interface exposes different API:
`Mono<SendMessage>`